tools/visual-tester/src/runner.js

```diff
@@ -217,6 +258,16 @@ module.exports = class Runner {
         const query = options.selector || this.globalOptions.selector;
         await page.waitForSelector(query);
         const selector = await page.$(query);
+
+        if (options.removeFromDom && options.removeFromDom !== '') {
+          for (const elToRemove of options.removeFromDom) {
+            await this._removeElementsFromDom(elToRemove, page);
+          }
+        }
+        if (options.hoverOver && options.hoverOver !== '') {
```

Comment:
```ts
async function hover(selector, page) { return (await page.$(selector)).hover() }
```

Or there could be even better way of doing it as 

```ts

it(URL, options, function(page, helpers) { })

```

Here helpers could already have the `page` object inside 

```ts
const helpers = {
  hover: (page) => (selector) => hover(selector, page)
}
```